config/routes.rb

```diff
@@ -4,14 +4,23 @@
   # get 'sessions/destroy'
   # get 'users/new'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  
+  resources :users do
+
+  #follow and unfollow
+  resources :relationships, only: [:create, :destroy]
+
   root 'pages#home'
 
   get '/about', to: 'pages#about'
   get '/contact', to: 'pages#contact'
   get '/signup', to: 'users#new'
 
```

Comment:
I think you forgot to make route/path for login and logout.
I receive error like this.
![image](https://user-images.githubusercontent.com/35232348/95849524-2b66d080-0d82-11eb-9c15-8c2f57e6705b.png)